client/my-sites/themes/theme-preview.jsx

```diff
@@ -19,37 +19,60 @@ export default React.createClass( {
 	propTypes: {
 		theme: React.PropTypes.object,
 		showPreview: React.PropTypes.bool,
+		showSecondaryButton: React.PropTypes.bool,
```

Comment:
Yeah, in fact that was the original idea, but there was an instance where it was useful to be able to pass the secondary button and decide separately if it should be shown (#6928). However, that instance is not in this PR, so maybe I should avoid premature optimization and just remove this prop.